en/mkdocs.yml

```diff
@@ -37,90 +37,67 @@ theme:
     accent: deep-orange
   custom_dir: theme/material
   logo: images/logo.svg
-  favicon: images/favicon.png
   highlightjs: true
   feature:
     tabs: true
   language: 'en'
   
 # Navigation
 nav:
-  - 'Home': index.md
-  - 'Getting Started':
-    - 'Introduction': getting-started/overview.md
+  - 'Introduction':
+    - 'Introduction': introduction/introduction.md
+    - 'Overview': introduction/overview.md
     - 'Architecture':
```

Comment:
Changes such as the 'home' icon and the structure change from "Introduction" to "Getting Started" section need to be there still but have been removed in this PR. Please check on these.

Reply:
Added changes to mkdocs.yml.